hugr/tests/cli.rs

```diff
@@ -91,3 +100,21 @@ fn test_bad_json(mut cmd: Command) {
         .failure()
         .stderr(contains("Error parsing input"));
 }
+
+#[rstest]
+fn test_bad_json_silent(mut cmd: Command) {
+    cmd.write_stdin(r#"{"foo": "bar"}"#);
+    cmd.args(["-", "-q"]);
+
+    cmd.assert()
+        .failure()
+        .stderr(contains("Error parsing input").not());
+}
+
+#[rstest]
+fn test_valid_silent(test_hugr_string: String, mut cmd: Command) {
+    cmd.arg("-");
+    cmd.write_stdin(test_hugr_string);
+
+    cmd.assert().success().stderr(contains(VALID_PRINT).not());
```

Comment:
not sure I like the default behaviour being silent. I was emulating e.g. mypy which prints a success message (we could also use colours for the messages 👀 )

Reply:
yeah happy with default at Info